scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -172,12 +172,61 @@
       (cc/write-compute-cluster conn {:compute-cluster/type :compute-cluster.type/kubernetes
                                       :compute-cluster/cluster-name compute-cluster-name}))))
 
+(defn get-bearer-token
+  [scoped-credentials]
+  (.refresh scoped-credentials)
+  (str "Bearer " (.getTokenValue (.getAccessToken scoped-credentials))))
+
+(def ^ScheduledExecutorService bearer-token-executor (Executors/newSingleThreadScheduledExecutor))
+
+(defn make-bearer-token-refresh-task
+  [api-client scoped-credentials]
+  (reify
+    Runnable
+    (run [_]
+      (try
+        (let [bearer-token (get-bearer-token scoped-credentials)]
+          (.setApiKey api-client bearer-token))
+        (catch Exception ex
+          (log/error ex "Error refreshing bearer token"))))))
+
+(defn make-api-client
+  [^String config-file base-path ^String google-credentials bearer-token-refresh-seconds verifying-ssl]
```

Comment:
What config file? Would be super useful to document here, (or better yet, the readme) how to get this all working.

Reply:
Added a docstring here, and a readme.